packages/components/src/internal/components/files/FileAttachmentForm.tsx

```diff
@@ -348,14 +348,14 @@ export class FileAttachmentForm extends React.Component<FileAttachmentFormProps,
 
     shouldRenderTemplateButton(): boolean {
         const { templateUrl } = this.props;
-        return templateUrl && templateUrl.length > 0 && !this.shouldShowPreviewGrid();
+        return templateUrl?.length > 0 && !this.shouldShowPreviewGrid();
     }
 
     renderTemplateButton() {
         return (
-            <Button bsStyle="info" title="Download Template" href={this.props.templateUrl}>
+            <a className="btn btn-info" title="Download Template" href={this.props.templateUrl} target="_blank">
```

Comment:
Suggest to add  rel="noopener noreferrer" whenever target blank, especially when the href is not a known static link. BTW, have you tried HTML5's new 'download' attribute?

Reply:
Add "noopener noreferrer", thank you. I didn't look into the HTML5 download attribute approach, but that would be good to consider in another PR.